proto/plan_common.proto

```diff
@@ -167,6 +169,8 @@ enum RowFormatType {
   UPSERT_AVRO = 11;
   DEBEZIUM_MONGO_JSON = 12;
   BYTES = 13;
+  DYNAMODB_JSON = 14;
+  DYNAMODB_CDC_JSON = 15;
```

Comment:
No need to add new variants in `RowFormatType`? @xiangjinwu

Reply:
updated.